README.md

```diff
@@ -1,8 +1,16 @@
-# Jazz - Instant sync
+# Jazz - Ship top-tier apps at high tempo
```

Comment:
Can you please change to "Build local-first apps"

Reply:
Done!